packages/flutter_tools/lib/src/compile.dart

```diff
@@ -481,7 +537,7 @@ class ResidentCompiler {
         }
       }
     }
-    return fileUri;
+    return null;
```

Comment:
Not sure why we are returning null if fileSystemRoots is null and we can't map file to package:-uri

Reply:
This is a helper function wrapped by `mapFilename` and `mapFileUri`.  Returning null tells the wrapper to return the original filename or URI